config-example/testvaults.txt

```diff
@@ -0,0 +1,7 @@
+# List of map elements to load for the map "test_vault.dmm", leave the file below this line blank to load all vaults.
```

Comment:
comment implies it louds only vaults but it seems to load all map elements

Reply:
it calls the proc get_map_element_objects if empty, and the default subtypes it searches for is /datum/map_element/vault, so it does only load vaults.